src/Plugin/FluentForms/AdCaptchaElements.php

```diff
@@ -1,30 +1,49 @@
 <?php
 
-namespace AdCaptcha\Plugin\FluentForms\AdCaptchaElement;
+namespace AdCaptcha\Plugin\FluentForms;
 
-use AdCaptcha\Widget\AdCaptcha\AdCaptcha;
-use AdCaptcha\Widget\Verify\Verify;
-
-class AdCaptchaElement extends \FluentForm\App\Services\FormBuilder\BaseFieldManager {
+use AdCaptcha\Widget\AdCaptcha;
+use AdCaptcha\Widget\Verify;
+use PhpParser\Error;
 
+class AdCaptchaElements extends \FluentForm\App\Services\FormBuilder\BaseFieldManager {
     /**
      * Constructor
      *
      * @return void
      */
-    public function __construct() {
-        parent::__construct(
-            'adcaptcha_widget',  // Changed to match adCAPTCHA response key
-            'adCAPTCHA',            // Title set to adCAPTCHA
-            [ 'captcha' ],
-            'advanced'
-        );
+    private $verify;
+    private $keyTest = 'adcaptcha_widget';
+    private $titleTest = 'adCAPTCHA';
+    public $printContentBaseFieldManager;
+
+    public function __construct($shouldInstantiateParent = true) {
+            if ($shouldInstantiateParent === true) {
+                parent::__construct('adcaptcha_widget',  
+                'adCAPTCHA',            
+                [ 'captcha' ],
+                'advanced');
+
+                $this->keyTest = $this->key;
+                $this->titleTest = $this->key;
```

Comment:
I don't like this naming. they aren't to do with testing.

Reply:
The name's I changed to widgetKey and widgetTitle.  
<img width="888" alt="Screenshot 2024-11-19 at 12 08 55" src="https://github.com/user-attachments/assets/a2df215c-a6c3-4e45-ac27-6efb80a78615">